backend/tests/seeder/test_conversation_demo_seeder.py

```diff
@@ -0,0 +1,54 @@
+import json
+import pytest
+from seeder.conversation_demo_seeder import (
+    create_client,
+    seed_chat_data,
+    get_last_user
+)
+from sqlalchemy import text
+from sqlmodel import Session
+
+
+@pytest.fixture
+def conversation_data():
+    json_path = "./seeder/conversation_demo.json"
+    with open(json_path, "r") as f:
+        data = json.load(f)
+    return data
+
+
+def test_create_client(session: Session):
+    create_client(session=session)
+    result = session.exec(text("SELECT COUNT(*) FROM \"client\""))
+    assert result.scalar() > 1
+
+
+def test_seed_chat_data(session: Session, conversation_data):
+    last_user = get_last_user(session=session)
+    assert last_user is not None
+
+    user_id = last_user.id
+    client_id = create_client(session=session)
+    assert client_id is not None
+
+    seed_chat_data(session=session, user_id=user_id, client_id=client_id)
+
+    # Verify chat sessions after seeding
+    chat_sessions = session.exec(
+        text("SELECT * FROM chat_session ORDER BY id DESC")
```

Comment:
please use sqlmodel command for consistency

Reply:
Updated mas @dedenbangkit